### PR TITLE
#4528 [Manager] Refactor the Topics page to use the Material-Table

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4528 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Topic/TopicListStyles.js [hunk 1]
```diff
@@ -18,12 +18,19 @@ import styled, { css } from 'styled-components';
 
 export const Wrapper = styled.div(
   ({ theme }) => css`
-    .filters {
-      margin-bottom: ${theme.spacing(3)}px;
+    .shared-topics {
```
Comment (konekoya) [inline ohara-manager/client/src/components/Topic/TopicListStyles.js:21]: Can be removed :)

File: ohara-manager/client/src/components/Topic/TopicListStyles.js [hunk 2]
```diff
@@ -18,12 +18,19 @@ import styled, { css } from 'styled-components';
...
+
+    .MuiPaper-root {
+      box-shadow: none;
+    }
 
-      .checkboxes {
-        margin-left: ${theme.spacing(3)}px;
-      }
+    .pipeline-link {
+      cursor: pointer;
```
Comment (konekoya) [inline ohara-manager/client/src/components/Topic/TopicListStyles.js:33]: Isn't the link already has this behavior?

File: ohara-manager/client/src/components/Topic/TopicListStyles.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/components/Topic/TopicListStyles.js]: Yes, I will remove it.

File: ohara-manager/client/src/components/Topic/TopicListStyles.js (hunk 2 above)
Comment (eechih) [reply ohara-manager/client/src/components/Topic/TopicListStyles.js:30]: 我也認為應該如此。不知為何放在 Material-Table 的 Column 中，pointer 的樣式就不見了？

Comment (konekoya) [reply ohara-manager/client/src/components/Topic/TopicListStyles.js:30]: Maybe their bug or something XD

**Pull request merged**